dataminer-overview/Legal/Commercial_Terms/Terms_partners.md

```diff
@@ -0,0 +1,61 @@
+---
+uid: Terms_partners
+---
+
+# Terms for partners
+
+When submitting the partnership application on Skyline Communications' website, by selecting the checkbox, you confirm that you have read, have understood, and agree to be bound by the Partner Program Terms below, including any linked policies. This action constitutes a legally binding acceptance of the terms under Belgian law.
+
+Skyline Communications nv\
+Ambachtenstraat 33\
+8870 Izegem\
+Belgium\
+Version 1.0, Tuesday, September 17, 2024
```

Comment:
Do we need to keep this here, or can it be removed?

Reply:
Can be removed